docs/plans/2026-05-09-pi-authentik-design.md

```
@@ -0,0 +1,135 @@
+# pi-authentik Design
+
+## Goal
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)